facebook/delphiFacebook/R/responses.R

```diff
@@ -457,6 +459,10 @@ create_complete_responses <- function(input_data, county_crosswalk)
     "V9", # added in Wave 7,
     "C14a", "C17a", "V2a", "V5a", "V5b", "V5c", "V5d", "V6", "D11", # added in Wave 8
     "C6a", "C8a_1", "C8a_2", "C8a_3", "C13b", "C13c", "V11", "V12", "V13", "V14_1", "V14_2", # added in Wave 10
+    "B10c", "B13 ", "C18a", "C18b", "C7a", "D12", "E4",
```

Comment:
```suggestion
    "B10c", "B13", "C18a", "C18b", "C7a", "D12", "E4",
```

Reply:
Is there a good way to compare this list against the QSF diff to verify that nothing is missing?

Reply:
This is taken directly from the diff (+ formatting changes, of course), so it should be exactly the same.